cristin-import/src/test/java/no/unit/nva/cristin/lambda/CristinEntryEventConsumerTest.java

```diff
@@ -175,6 +176,23 @@ void shouldPersistCristinIdInFileNamedWithPublicationIdentifier() throws IOExcep
         assertDoesNotThrow(() -> s3Driver.getFile(expectedErrorFileLocation));
     }
 
+    @Test
+    void shouldPersistNviDataWhenCristinObjectContainsNviData() throws IOException {
+        var cristinObject = CristinDataGenerator.randomObjectWithReportedYear(2011);
+        var eventBody = createEventBody(cristinObject);
+        var sqsEvent = createSqsEvent(eventBody);
+        var publications = handler.handleRequest(sqsEvent, CONTEXT);
+        var actualPublication = publications.get(0);
+        var expectedFileNameStoredInS3 = actualPublication.getIdentifier().toString();
+
+        var expectedTimestamp = eventBody.getTimestamp();
+        var expectedFileLocation = NVI_FOLDER
+                                            .addChild(timestampToString(expectedTimestamp))
+                                            .addChild(expectedFileNameStoredInS3);
+
+        assertDoesNotThrow(() -> s3Driver.getFile(expectedFileLocation));
```

Comment:
This is a bit on the thin side. I would like some asserts that you can parse the data from the report and that the data contains what you except.